cmd/nodecmd/validateSubnet.go

```diff
@@ -75,30 +71,42 @@ func parseSubnetSyncOutput(filePath string, printOutput bool) (string, error) {
 	return "", errors.New("unable to parse subnet sync status")
 }
 
-func addNodeAsSubnetValidator(nodeID string, network models.Network) error {
+func addNodeAsSubnetValidator(nodeID string, network models.Network, currentNodeIndex, nodeCount int) error {
 	ux.Logger.PrintToUser("Adding the node as a Subnet Validator...")
 	if err := subnetcmd.CallAddValidator(subnetName, nodeID, network); err != nil {
 		return err
 	}
-	ux.Logger.PrintToUser("Node successfully added as Subnet validator!")
+	ux.Logger.PrintToUser(fmt.Sprintf("Node %s successfully added as Subnet validator! (%d / %d)", nodeID, currentNodeIndex+1, nodeCount))
+	ux.Logger.PrintToUser("======================================")
 	return nil
 }
 
-func getNodeSubnetSyncStatus(blockchainID, clusterName string, printOutput, errOnValidating bool) (bool, error) {
-	ux.Logger.PrintToUser("Checking if node is synced to subnet ...")
+func getNodeSubnetSyncStatus(blockchainID, clusterName, hostAlias string, statusOutput, errOnValidating bool) (bool, error) {
```

Comment:
comment what this function checks and it what cases
maybe it would be better for this function to check subnet status against a expected status given as funtion param, where status cmd pass in validating, and this command pass in syncing, and make the external function fail with error. it seems to me that the logic from two different usages is beign put inside here and may be better to have outside, in the callers